pr-data.csv

```diff
@@ -3646,8 +3646,8 @@ https://github.com/schemacrawler/SchemaCrawler,18d6c5fb79c8e27cdeafc317ee63b2783
 https://github.com/schemacrawler/SchemaCrawler,18d6c5fb79c8e27cdeafc317ee63b27837aa6fec,schemacrawler-utility,us.fatehi.utility.test.html.TableCellTest.td2,ID,InspiredAFix,https://github.com/schemacrawler/SchemaCrawler/pull/525,
 https://github.com/seata/seata,d334f85395887193f72fa57daea906a73a2da34e,common,io.seata.common.util.CollectionUtilsTest.encodeMap,ID,Accepted,https://github.com/seata/seata/pull/1802,
 https://github.com/seata/seata,d334f85395887193f72fa57daea906a73a2da34e,tm,io.seata.tm.api.transaction.TransactionInfoTest.testFieldGetSetFromJson,ID,Deleted,,https://github.com/seata/seata/commit/6c0c75939dedbf19c4a21a8ac7aa0fb98879a8fd
-https://github.com/sinch/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,,,
-https://github.com/sinch/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteIOS,ID,,,
+https://github.com/sinch/robovm,ef091902377c00dc0fb2db87e8d79c8afb5e9010,compiler,org.robovm.compiler.config.ConfigTest.testWriteConsole,ID,Opened,https://github.com/robovm/robovm/pull/1272,
```

Comment:
This PR link should be on `robovm/robovm` not `sinch/robovm`. In fact, these forked repos like `sinch/robovm` should not even be in `pr-data.csv`. Please remove these entries (as in delete them altogether not just add `Deleted`)?

Reply:
Thank you, updated!